media/transcoder/job_template_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.cloud import resource_manager
+
+import create_job_template
+import delete_job_template
+import get_job_template
+import list_job_templates
+
+location = "us-central1"
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+template_id = "my-python-test-template"
+
+
+def test_template_operations(capsys):
+
+    # Enable the following APIs on the test project:
+    # *   Transcoder API
+    # *   Cloud Resource Manager API (needed for project number translation)
+
+    client = resource_manager.Client()
+    project = client.fetch_project(project_id)
```

Comment:
This requires an organization role that most accounts are unlikely to have. Perhaps this should just be an environment variable (if it is needed, see comment below).

Reply:
Removed dependency on the Cloud Resource Manager API in favor of setting an environment variable.